app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -1220,7 +1221,10 @@ export class Dispatcher {
       }
 
       this.statsStore.recordRebaseSuccessWithoutConflicts()
-      await this.completeMultiCommitOperation(repository, commits.length)
+      await this.completeMultiCommitOperation(
+        repository,
+        result === RebaseResult.AlreadyUpToDate ? 0 : commits.length
```

Comment:
Not super happy about this to differentiate between "just successful" and "successful but did nothing" banners, but I didn't see a better way to pass that info along.

Reply:
+1226

Reply:
I am not sure `completeMultiCommitOperation` makes sense here. 

It:
1. Gets the banner
2. Sets force push list if need be
3. Records Operation Successful. -> I wouldn't say that is what this is... this is a operation didn't happen :D 
4. calls `endMultiCommitOperation` (nulls out state)
5. Refreshes Repository.

I think we only need:
2, 4, and maybe 5. 

What do you think of either:

Simply, 
```
    this.setBanner({... banner details ...}
    this.endMultiCommitOperation(repository)
    await this.refreshRepository(repository)
```

Or... making another multiCommitOperation "result path" for lack of better work, that is a function like `completeMultiCommitOperation` but entitled something like `multiCommitOperationAlreadyUpToDate`. This, only makes senes to me if we expect some other one to have a similar scenario. We do have something like this in Cherry-pick... called `CherryPickResult.UnableToStart`.. but isn't the exactly same as no success banner..

Other thoughts, we could add a stat for this.